environments/skeleton/{{cookiecutter.environment}}/terraform/variables.tf

```diff
@@ -136,3 +137,9 @@ variable "k3s_token" {
     description = "K3s cluster authentication token, set automatically by Ansible"
     type = string
 }
+
```

Comment:
Nope, we don't want to define this globally, just needs to be defined in the `compute` variable.